scripts/check_blocking_repos.py

```diff
@@ -51,7 +57,14 @@ def is_released(repo, dist_file):
 if i == 0:
     print('No previous distribution found.')
     exit(-1)
-prev_distro_key = valid_distro_keys[i - 1]
+
+if args.comparison:
```

Comment:
This case doesn't work if `--rosdistro` is the first distro in the list. Please modify the check from line 57 accordingly.